src/System.Globalization/tests/CompareInfo/CompareInfoTests.IndexOf.LastIndexOf.cs

```diff
@@ -0,0 +1,494 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I would call this file just CompareInfoTests.IndexOf.cs it is kind implicit this test all indexof APIs family